README.md

```diff
@@ -120,8 +121,13 @@ If you find an issue with this package and have a fix, please feel free to open
 
 ## Testing
 
-If you contribute to this package you can run the tests using `yarn test`. This package also includes a sample application for manual testing. First open a seperate terminal and run `yarn watch`, this will ensure the Typescript files in the src directory are compiled to Javascript in the lib directory. Navigate to `src/sample`, here you can edit `index.ts` to your liking and test your contributions manually. Run `npx cdk --app lib/sample/index.js <CDK Command>`, replacing `<CDK Command>` with common CDK commands like `synth`, `diff`, or `deploy`. 
-* Note, if you receive "... is not authorized to perform: ..." you may also need to authorize the commands with your AWS credentials.
+If you contribute to this package you can run the tests using `yarn test`. This package also includes a sample application for manual testing. First open a seperate terminal and run `yarn watch`, this will ensure the Typescript files in the src directory are compiled to Javascript in the lib directory. Navigate to `src/sample`, here you can edit `index.ts` to your liking and test your contributions manually. Run `npx cdk --app lib/sample/index.js <CDK Command>`, replacing `<CDK Command>` with common CDK commands like `synth`, `diff`, or `deploy`.
+
```

Comment:
I am curious if the sample app gave you any troubles? For instance did you have any issues deploying because you were not in the root directory? I'm thinking about updating this section to be a bit more clear with the instructions. I can do that myself in a separate PR I just wanted to hear your experience.

Reply:
Yeah, I had to add a `cdk.json` file to be able to run `cdk synth` from within the sample project:

```
{
  "app": "npx ts-node index.ts"
}
```

I also had to update the path where the Lambda code was coming from:

```
      code: lambda.Code.fromAsset("./lambda"),
```

I also tried the updated instructions in #24 and they worked great. The only additional instruction I would maybe add is to run `npx projen` to regenerate the `package.json` when making any changes to the dependencies. This instruction is there in the `package.json` file but would be helpful to have in the README as a step before `yarn watch`.